config/environments/development.rb

```diff
@@ -63,6 +63,8 @@
     # include paths with any of these substrings in the stack trace,
     # even if they are not in your main app
     # Bullet.stacktrace_includes = [ 'your_gem', 'your_middleware' ]
+    Bullet.add_whitelist :type => :n_plus_one_query, :class_name => "Slide", :association => :tag
```

Comment:
__[rubocop]__
* Style/HashSyntax: Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)
* Style/HashSyntax: Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Style/HashSyntax: Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)